server/models/Profile.js

```diff
@@ -0,0 +1,41 @@
+const mongoose = require("mongoose");
+
+const reqString = {
```

Comment:
Nit: Spelling this out as `requiredString` would be better than `reqString` as `reqString` seems to mean `request string`.